src/features/nft/addNFT/approvedStep/ApprovedStep.tsx

```diff
@@ -56,8 +57,26 @@ export default function ApprovedStep({
             <InfoPair title='Patel rareness score' value='67%' />
             <InfoPair title='Internet rareness score' value='99%' />
             <InfoPair title='NSFW' value='100%' />
+            <div>
+              <div className='font-medium text-gray-71 mb-3'>
+                Thumbnail preview
+              </div>
+              <div className='w-48 h-48'>
+                {croppedImage && (
+                  <img
+                    src={croppedImage.src}
+                    className='rounded w-full h-full'
+                  />
+                )}
+              </div>
+              {croppedImage?.error && (
+                <div className='text-sm text-error font-medium mt-3'>
+                  Error text error text
```

Comment:
We can't have any errors on ApprovedStep because if something wrong user never should be here, remove this error pls

Reply:
fixed